endpoint/endpoint.go

```diff
@@ -408,8 +412,9 @@ type ReceiverConfig struct {
 
 	RootWithoutClientComponent *zfs.DatasetPath // TODO use
 	AppendClientIdentity       bool
-
-	UpdateLastReceivedHold bool
+	InheritProperties          []zfs.Property
+	OverrideProperties         map[zfs.Property]string
```

Comment:
we'll need to `ReceiverConfig.copyIn` both `InheritProperties` and `OverrideProperties` (this is a method right below this one, please put a comment in here to make this sufficiently clear for future modifications)

Reply:
I'm unsure why we need to do that, although I can imagine a few possible reasons. Do you have a  suggestion for such a comment?

Reply:
we do the copy-in to ensure that the `ReceiverConfig` object is self-contained.

I guess the comment should be on the entire `ReceiverConfig` struct:

```
+ // NOTE: when adding members to this struct, remember
+ //       to add them to `ReceiverConfig.copyIn()`
  type ReceiverConfig struct {
```